index.html

```diff
@@ -46,22 +45,19 @@ <h2 class="desktop">Hi there. I'm Alfred</h2>
                 <li class="Angle social"><a href="#"><img src="images/angellist.svg" alt="Angle"></a></li>
             </ul>
             <div class="drop-down"> <img src="images/dropdown.png" alt=""></div>
-
         </section>
         <!-- card one -->
         <section class="top-card-one">
             <h2>My Recent <br> Works</h2>
             <div class="bd">
```

Comment:
- [ ] [OPTIONAL] Be aware of the repetition of this `<h2>` tag. This should be used as the header of the section once, not in all cards. Take a look at [the Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

Reply:
@leonardodiasb 
Thanks for the review, I will definitely implement